### DISCUSSION TITLE
Bilsem Bilişim Proje Fikirleri

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus.github.io#94] [Fikirler]
Bilim ve Sanat Merkezinde okuyan öğrenciler için bilişim dersinde geliştirilebilecek olan proje fikirlerini bu başlık altına ekleyiniz. Proje fikrini yazarken, projenin adı, amacı, yöntemi ve sonucu gibi detaylı yazabileceğiniz gibi sadece projenin fikrini de kısaca yazabilirsiniz.

**Comment by sonsuzus** [comment]
* Herhangi bir programlam dilinin araştırılması ve özelliklerinin sunulması
* Herhangi bir programlama dilinde bir veya daha fazla modülün, kütüphanenin kullanılması ve eksikliklerinin tespiti
* Çok nadir kullanılan ilginç programlama dillerinin tespiti ve araştırılması
* Yapayzeka botları için promtlar

**Comment by omersalih177** [comment]
Proje Adı: Olumsuz Durum Algılayıcı
Amacı: Uygulamanın insanların o anki olumsuz hallerini algılayıp çözüm üretmesi
Yöntem: Uygulama insandaki nabız atışı ve stresi algılar ve çözüm önerileri sunar.
Sonuç: Olumsuz bir durum yaşadığınızda en iyi şekilde çözüm buluyorsunuz.

**Comment by omersalih177** [comment]
Proje Adı: Kitaplık Düzenleyici
Amacı: Kitapları düzenleyip insanlara zaman kazandırmak
Yöntemi: Kitaplığınızın arkasındaki düğmeyi çeviriyorsunuz ve mekanik kollar kitapları düzenli hale getiriyor. Bunu kolların daha hızlı yapabilmesi için ayarını arttırabilirsiniz.
Sonuç: İnsanlar kitaplıklarıyla uğraşmak zorunda kalmıyor.
Sonuç:

**Comment by omersalih177** [comment]
Projenin Adı: Dikkatin Kaybolmasın Gözlüğü
Amacı: Öğrencilerin ders sırasında dikkatlerinin kaybolmaması
Yöntem: Öğrenci okulda veya evde ders çalışırken veya soru çözerken dikkatlerinin başka yöne gitmemesi için kendilerinde bulunan gözlükler titreşimler yayar.
Sonuç: Öğrenciler dikkatleri kaybolmadığı için daha sağlıklı ders çalışırlar.

**Comment by omersalih177** [comment]
Projenin Adı: Kuraklık Çözüm İster
Amacı: Kuraklığın olduğu bölgelerde yaşayan insanların su bulması
Yöntem: Havadaki nemi suya dönüştüren bir düzenek haznesinde nemi soğutup suya dönüştürür, bu sayede insanlar bu suyu kullanabilir.
Sonuç: Kuraklığa karşı geçici bir önlem alınmış olur.

**Comment by omersalih177** [comment]
Projenin Adı: Suyu Doğru Kullan
Amacı: İnsanların su tasarrufu konusunda bilinçlenmesini sağlar.
Yöntem: İnsanlara bir gün boyunca belirli bir miktarda su kullanım hakkı verilmesi, bu hakkın dışında suyun tüketilememesi.
Sonuç: İnsanların suyu dengeli kullanma bilincinin geliştirilmesi.
